build_scripts/installer-version.py

```diff
@@ -21,7 +21,7 @@ def main():
         dev_release_number = ""
     elif len(version) == 4:
         smc_patch_version = version[2]
-        dev_release_number = "." + version[3]
+        dev_release_number = "-" + version[3]
```

Comment:
Did the old versioning scheme not work anymore with the new builders or what was the reason for this change?